### PR TITLE
Additional Refactoring of lib.plot [Part 2: conversion to package]

### PR DESCRIPTION
**Pull request by dougthor42**
This PR is primarily to convert `wx.lib.plot` to its own package.

This will make it much easier to maintain and expand in the future.

+ ~~[ ] Add unit tests~~
+ [x] Adjust location of changelog
+ [x] Ensure that things are backwards compatible.
+ [x] Separate out example from source.
+ [x] Add a much more simple example

### CODE REVIEW

File: wx/lib/plot/__init__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+wx.lib.plot
+===========
+
+This is a simple plotting library for the wxPython Phoenix project.
+
+"""
+__version__ = "0.0.1"
+__updated__ = "2016-07-05"
+
+# For those who still use ``from pakcage import *`` for some reason
```
Comment (RobinD42) [inline wx/lib/plot/__init__.py]: Typo.  (Personally, I usually swap the a and the k 😃  )

**Comment by dougthor42** [comment]
Just FYI, I'll rebase this to master after #98 gets merged. This will make viewing the diffs much easier.

And thanks for note about the typo. I always do that one... also "improt" :laughing:

**Comment by dougthor42** [comment]
Ready for review.

**Comment by RobinD42** [comment]
LGTM. Merging.

**Pull request merged**